example/App.vue

```diff
@@ -117,7 +117,11 @@ export default class App extends Vue {
         this.offset += 1;
 
         // Wait between each response just to see the loader
-        return new Promise((resolve) => setTimeout(() => resolve(false), 2000));
+        return Promise.resolve(false);
+    }
+
+    pullDataWithDelay(): Promise<boolean> {
+        return new Promise((resolve) => setTimeout(() => resolve(this.pullData()), 2000));
```

Comment:
Shouldn't this be configurable instead of an arbitrary delay?

Reply:
It's only for the demo website :smile:

Reply:
ooops :see_no_evil: